pkg/aws_config_client/completer.go

```diff
@@ -252,10 +292,17 @@ func (c *completer) Complete(out *ini.File) error {
 	if err != nil {
 		return err
 	}
+	if len(profiles) == 0 {
+		logrus.Info("No AWS profiles configured")
+		return nil
+	}
 
-	err = c.writeAWSProfiles(out, region, profiles)
+	newAWSProfiles, err := c.assembleAWSConfig(region, profiles)
 	if err != nil {
 		return err
 	}
-	return nil
+
+	mergedConfig, err := c.mergeConfigs(newAWSProfiles, base)
+	writeCode, err := mergedConfig.WriteTo(w)
+	return errors.Wrapf(err, "Could not write new aws config. WriteCode: %d", writeCode)
```

Comment:
what is `writeCode`?

Reply:
I misinterpreted that variable (actually means # of bytes written). Will be deleting references to it